core/src/identity/auth/oauth/oauth.service.ts

```diff
@@ -19,7 +19,7 @@ export class OauthService {
 
     async loginDiscord(user: AuthPayload): Promise<AccessToken> {
         return {
-            access_token: this.jwtService.sign(user, {expiresIn: "6h"}),
+            access_token: this.jwtService.sign(user, {expiresIn: "1m"}),
```

Comment:
This seems like a great spot for a config file value

Reply:
I have a ticket in clickup for exactly this, but we deemed it a pretty low priority. I can always sneak that in here.

Reply:
Fixed in latest commit